test/dialect/mysql/test_query.py

```diff
@@ -255,3 +261,47 @@ def test_any_literal(self, connection):
         stmt = select(4 == any_(select(stuff.c.value).scalar_subquery()))
 
         is_(connection.execute(stmt).scalar(), True)
+
+
+class ComputedTest(fixtures.TestBase):
+    __only_on__ = "mysql >= 5.7", "mariadb"
+    __backend__ = True
+
+    @combinations(
+        (True),
+        (False),
+        (None),
+        ("unset"),
+        argnames="nullable",
+    )
+    def test_column_computed_for_nullable(self, connection, nullable):
+        """test #10056
+
+        we want to make sure that nullable is always set to True for computed
+        column as it is not supported for mariaDB
+        ref: https://mariadb.com/kb/en/generated-columns/#statement-support
+
+        """
+        m = MetaData()
+        kwargs = {"nullable": nullable} if nullable != "unset" else {}
+        t = Table(
+            "t",
+            m,
+            Column("x", Integer),
+            Column("y", Integer, Computed("x + 2"), **kwargs),
+        )
+        if connection.engine.dialect.name == "mariadb" and nullable in (
+            False,
```

Comment:
**mike bayer** (zzzeek) wrote:

hm it looks like "nullable=None" means, on SQL Server at least, "dont write any NULL clause at all".  not sure what it does for mariadb, but it's not really a use case here so this is fine.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4869